test/e2e/page-objects/pages/home/homepage.ts

```diff
@@ -180,11 +180,11 @@ class HomePage {
   /**
    * Checks if the expected balance is displayed on homepage.
    *
-   * @param expectedBalance - The expected balance to be displayed. Defaults to '0'.
+   * @param expectedBalance - The expected balance to be displayed. Defaults to '25'.
    * @param symbol - The symbol of the currency or token. Defaults to 'ETH'.
    */
   async check_expectedBalanceIsDisplayed(
-    expectedBalance: string = '0',
+    expectedBalance: string = '25',
```

Comment:
we set 25 as default as it's the default value set in ganache class. It's easier to have this as 25 and change it in the places we want